central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -169,6 +168,11 @@ type flowStoreImpl struct {
 	partitionName string
 }
 
+type srcDstEntityIds struct {
+	srcId	string
+	dstId	string
+}
```

Comment:
Not strictly needed, but does make it so that we can easily get the src ID when needed and the dst ID when needed.

Reply:
Moved to another PR.